src/main/java/org/sagebionetworks/bridge/exporter/record/BridgeExporterRecordProcessor.java

```diff
@@ -193,6 +210,15 @@ public void processRecordsForRequest(BridgeExporterRequest request) throws IOExc
 
             // We made it to the end. Set the success flag on the task.
             setTaskSuccess(task);
+
+            // finally modify export time table in ddb
+            List<String> studyIdsToUpdate = exportHelper.bootstrapStudyIdsToQuery(request);
+            DateTime endDateTime = exportHelper.getEndDateTime(request);
+            if (!studyIdsToUpdate.isEmpty() && endDateTime != null) {
+                for (String studyId: studyIdsToUpdate) {
+                    ddbExportTimeTable.putItem(new Item().withPrimaryKey(STUDY_ID, studyId).withNumber(LAST_EXPORT_DATE_TIME, endDateTime.getMillis()));
+                }
+            }
```

Comment:
We should move this into its own helper class, so that RecordProcessor isn't a big monolithic class that does everything.

Also, what about error handling? What happens when one of these puts fail?